app/graphql/types/interfaces/rules_preload.rb

```diff
@@ -68,12 +68,9 @@ def initialize_rules_context(rules, rule_results, args = {})
 
       def initialize_rule_references_context(rule_results)
         rule_ids = rule_results.pluck(:rule_id)
-        grouped_rules_references = ::RuleReferencesRule.distinct.where(
-          rule_id: rule_ids
-        ).group_by(&:rule_id)
-        grouped_rules_references.each do |rule_id, references|
-          context[:"rule_references_#{rule_id}"] =
-            references.pluck(:rule_reference_id)
+
+        RuleReferencesContainer.where(rule_id: rule_ids).find_each do |reference|
+          context[:"rule_references_#{container.rule_id}"] = reference.rule_references
```

Comment:
What is `container` on this line? I don't see it being defined anywhere but maybe I'm missing it

Reply:
Yeah, this was a bug :see_no_evil: